<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Эмулятор высокоомной установки</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>bug_32.png</normaloff>bug_32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0" rowspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>530</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="TabReport">
       <attribute name="title">
        <string>Отчёт</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="MemoReport">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabLog">
       <attribute name="title">
        <string>Лог</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="MemoLog"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="MultimeterBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>346</width>
        <height>82</height>
       </size>
      </property>
      <property name="title">
       <string>Мультиметр</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="MultimeterLabel">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>0.000000000000</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="ComparatorCaptionLabel">
         <property name="text">
          <string>Компаратор:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="ComparatorLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="BalanceBox">
      <property name="minimumSize">
       <size>
        <width>357</width>
        <height>362</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="title">
       <string>Уравновешивание</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QCheckBox" name="BalanceAdaptivePauseCheckBox">
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Адаптивная пауза</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="4" colspan="2">
        <widget class="QGroupBox" name="CoilsParamGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>107</height>
          </size>
         </property>
         <property name="title">
          <string>Катушки</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="SourceVoltageLabel">
            <property name="text">
             <string>Источник</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="3">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QDoubleSpinBox" name="SourceVoltageSpinBox">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>200.009999999999991</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="CoilEtalonSpinBox">
              <property name="decimals">
               <number>7</number>
              </property>
              <property name="maximum">
               <double>120.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="CoilCheckedSpinBox">
              <property name="decimals">
               <number>7</number>
              </property>
              <property name="maximum">
               <double>120.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="SourceVoltageUnitsLabel">
            <property name="text">
             <string>В</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="CoilEtalonCaptionLabel">
            <property name="text">
             <string>Эталон</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="CoilEtalonUnitsLabel">
            <property name="text">
             <string>МОм</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="CoilCheckedCaptionLabel">
            <property name="text">
             <string>Поверяемая</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="CoilCheckedUnitsLabel">
            <property name="text">
             <string>МОм</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="CalibreSubtractBiasCheckBox">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Вычесть смещение</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="CalibreCheckingWeight">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Проверить вес</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" rowspan="2">
        <widget class="QGroupBox" name="BalanceUseGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Использовать</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="BalanceUseMultimeterButton">
            <property name="text">
             <string>Мультиметр</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="BalanceUseComparatorButton">
            <property name="text">
             <string>Компаратор</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="2" rowspan="2">
        <widget class="QGroupBox" name="ActionGroupBox">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>216</height>
          </size>
         </property>
         <property name="title">
          <string>Действие</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QRadioButton" name="CalibreRadioButton">
            <property name="text">
             <string>Калибровка</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="BalanceRadioButton">
            <property name="text">
             <string>Уравновешивание</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="BalanceEmulateRadioButton">
            <property name="text">
             <string>Эмуляция уравновешивания</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="BufferTestRadioButton">
            <property name="text">
             <string>Проверка буфера</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="R2RScanRadioButton">
            <property name="text">
             <string>Проверка ЦАП</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="R2RScanEmulateRadioButton">
            <property name="text">
             <string>Эмуляция проверки ЦАП</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QRadioButton" name="OneMeasRadioButton">
            <property name="text">
             <string>Одно измерение</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="ActionButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Сделать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="BalancePauseLabel">
           <property name="text">
            <string>Пауза</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="BalancePauseSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="BalancePauseUnitsLabel">
           <property name="text">
            <string>с</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="NPLCEdit">
           <property name="text">
            <string>NPLC</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="NPLCSpinBox">
           <property name="minimum">
            <double>0.020000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="BalanceMeasCntLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Измерений</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="BalanceMeasCntSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="LSBCountLabel">
           <property name="text">
            <string>Мл. разр.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="LSBCountSpinBox"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="LSBMeasCountLabel">
           <property name="text">
            <string>Изм. мл. разр.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="LSBMeasCountSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="BalanceExpCntLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Опытов</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QSpinBox" name="BalanceExpCntSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="BufferTestDigitsLabel">
           <property name="text">
            <string>Тест буф., разр.</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QSpinBox" name="BufferTestDigitsSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="R2RScanStartDigitLabel">
           <property name="text">
            <string>Пров. ЦАП, старт</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QSpinBox" name="R2RScanStartDigitSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QLabel" name="R2RScanDigitsLabel">
           <property name="text">
            <string>Пров. ЦАП, разр.</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QSpinBox" name="R2RScanDigitsSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>22</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0" rowspan="3">
       <widget class="QGroupBox" name="CodeBox">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Код</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QSpinBox" name="CodeSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="SetCodeButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Записать код</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="TestLedCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Светодиод</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="AbortButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="SaveSettingsButton">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить настройки</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="ClearButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%p%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
